css/properties/scroll-behavior.json

```diff
@@ -6,17 +6,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/scroll-behavior",
           "support": {
             "chrome": {
-              "version_added": "61",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Enable experimental web platform features",
-                  "value_to_set": "true"
-                }
-              ]
+              "version_added": "61"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "61"
```

Comment:
`webview_android` should also be `"61"`.

Reply:
You are absolutely right. I've changed it.